### PR TITLE
get mailbox status once for all boxes

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@jancborchardt

**Comment by jancborchardt** [comment]
:+1: everything still works

File: lib/account.php [hunk 1]
```diff
@@ -123,6 +123,7 @@ public function createMailbox($mailBox) {
 		$conn->createMailbox($mailBox);
 		$this->mailboxes = null;
 
+//		$mailBox = \Horde_Imap_Client_Mailbox::get($mailBox, false)->utf7imap;
```
Comment (clementhk) [inline lib/account.php]: I wonder where this is from, as I thought I've removed all utf7imap related code.

Comment (clementhk) [inline lib/account.php]: Ohh this is a new line, not commenting out old lines..
We should remove this as it makes no sense, I think.

Comment (DeepDiver1975) [inline lib/account.php]: indeed

Comment (jancborchardt) [inline lib/account.php]: Just remove it ;) pull at https://github.com/owncloud/mail/pull/599

**Pull request merged**